### PR TITLE
Updates to the Aurora Serverless v2 tutorial

### PR DESCRIPTION
**Pull request by cdaveaws**
This tutorial fixes a few issues that were identified after the tutorial was
committed. This also includes examples for using both PostgreSQL and MySQL.

Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 1]
```diff
@@ -51,6 +48,13 @@ The first method uses [Amazon EKS Workshop](https://www.eksworkshop.com/010_intr
 
 To manage an Aurora Serverless v2 cluster from Kubernetes / Amazon EKS, you will need to install the ACK for RDS service controller. You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart).
 
+Define environment variables
+
+```
+export SERVICE=rds
+export RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:55]: nit: The backtick syntax is considered deprecated
```suggestion
export RELEASE_VERSION=$(curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
```

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 2]
```diff
@@ -63,8 +67,8 @@ You can now use the Helm chart to deploy the ACK service controller for Amazon R
 For example, to specify that the RDS API calls go to the `us-east-1` region, you can deploy the service controller with the following command:
 
 ```bash
-helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/rds-chart \
-  --version=v0.0.27 --generate-name --set=aws.region=us-east-1
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:70]: `$AWS_REGION` is not defined anywhere above

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 3]
```diff
@@ -51,6 +48,14 @@ The first method uses [Amazon EKS Workshop](https://www.eksworkshop.com/010_intr
 
 To manage an Aurora Serverless v2 cluster from Kubernetes / Amazon EKS, you will need to install the ACK for RDS service controller. You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart).
 
+Define environment variables
+
+```
+export SERVICE=rds
+export RELEASE_VERSION=$(curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
```
Comment (jkatz) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:55]:
```suggestion
export RELEASE_VERSION=$(curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
```
```suggestion
RELEASE_VERSION=$(curl -sL "https://api.github.com/repos/aws-controllers-k8s/${SERVICE}-controller/releases/latest" | grep '"tag_name":' | cut -d'"' -f4)
```

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 4]
```diff
@@ -63,8 +68,8 @@ You can now use the Helm chart to deploy the ACK service controller for Amazon R
 For example, to specify that the RDS API calls go to the `us-east-1` region, you can deploy the service controller with the following command:
 
 ```bash
-helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/rds-chart \
-  --version=v0.0.27 --generate-name --set=aws.region=us-east-1
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
```
Comment (jkatz) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:71]:
```suggestion
aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
```
```suggestion
aws ecr-public get-login-password --region "${AWS_REGION}" | helm registry login --username AWS --password-stdin public.ecr.aws
```

Comment (cdaveaws) [reply docs/content/docs/tutorials/aurora-serverless-v2.md:71]: When authenticating to a public registry, always authenticate to the us-east-1 Region when using the AWS CLI. https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 5]
```diff
@@ -51,6 +48,14 @@ The first method uses [Amazon EKS Workshop](https://www.eksworkshop.com/010_intr
 
 To manage an Aurora Serverless v2 cluster from Kubernetes / Amazon EKS, you will need to install the ACK for RDS service controller. You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart).
 
+Define environment variables
+
+```
+export SERVICE=rds
+RELEASE_VERSION=$(curl -sL "https://api.github.com/repos/aws-controllers-k8s/${SERVICE}-controller/releases/latest" | grep '"tag_name":' | cut -d'"' -f4)
+export ACK_SYSTEM_NAMESPACE=ack-system
+export AWS_REGION=$(curl -s 169.254.169.254/latest/dynamic/instance-identity/document | jq -r '.region')
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:57]: I understand that this tutorial was written to be used within a ec2-based environment, but I am a little hesitant that we recommend using the IMDS URL in tutorials. My preference would be that we ask users which region they would like to deploy resources, and then these scripts can be a little more re-usable for people using them outside Cloud9.

File: docs/content/docs/tutorials/aurora-serverless-v2.md [hunk 6]
```diff
@@ -137,7 +143,11 @@ kubectl create secret generic -n "${APP_NAMESPACE}" ack-creds \
   --from-literal=password="${RDS_DB_PASSWORD}"
 ```
 
-You can now create an Aurora Serverless v2 cluster:
+
+You can now create an Aurora Serverless v2 cluster for both the PostgreSQL and
+MySQL database engines. The example below uses the PostgreSQL engine. To use
+MySQL, set `ENGINE_TYPE` to `aurora-mysql` and `ENGINE_VERSION` to `8.0`.
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/aurora-serverless-v2.md:149]: nit: Worth linking to a page where users can find the latest version of the engines, so you aren't forced to update this tutorial with every release?

**Review by RedbackThomson — COMMENTED** [review]
I have left 2 nits. Otherwise I'm fine with these changes

File: docs/content/docs/tutorials/aurora-serverless-v2.md (hunk 6 above)
Comment (cdaveaws) [reply docs/content/docs/tutorials/aurora-serverless-v2.md]: fixed

File: docs/content/docs/tutorials/aurora-serverless-v2.md (hunk 5 above)
Comment (cdaveaws) [reply docs/content/docs/tutorials/aurora-serverless-v2.md:57]: do you mean we provide scripts to build EKS inline here? we can try that in the next release?

Comment (jkatz) [reply docs/content/docs/tutorials/aurora-serverless-v2.md:57]: Perhaps to say another way, can we just return to hardcoding the region in? I think users can set the region manually (similar to what we do in the RDS tutorial)

**Comment by RedbackThomson** [comment]
/approve
Will wait for @jkatz to sign-off

**Comment by jkatz** [comment]
@RedbackThomson Gave the `lgtm` :+1:

**Pull request merged**